src/commands/mod.rs

```diff
@@ -539,6 +541,82 @@ struct NewArgs {
     insert_before: bool,
 }
 
+/// Move the current working copy commit to the next child revision in the
+/// repository.
+///
+///
+/// The command moves you to the next child in a linear fashion.
+///
+///
+/// D        D  @
+/// |        | /
+/// C  @ =>  C
+/// | /      |
+/// B        B
+///
+///
+/// If `--edit` is passed, it will move you directly to the child
+/// revision.
+///
+///
+/// D        D
+/// |        |
+/// C        C
+/// |        |
+/// B  @  => @
+/// | /      |
+/// A        A
+// TODO(#NNN): Handle multiple child revisions properly.
+#[derive(clap::Args, Clone, Debug)]
+#[command(verbatim_doc_comment)]
+struct NextArgs {
+    /// How many revisions to move forward. By default advances to the next
+    /// child.
+    #[arg(default_value = "1")]
+    amount: usize,
```

Comment:
nit: could be `u64` to eliminate integer conversion?

Reply:
done